bears/go/GoMetaLintBear.py

```diff
@@ -0,0 +1,66 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='gometalinter',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.go):'
+                     r'(?P<line>\d+):'
+                     r'(?P<column>\d*):'
+                     r'(?P<severity>[a-z]+): '
+                     r'(?P<message>.*) '
+                     r'(?P<additional_info>\([a-z]+\))')
+class GoMetaLintBear:
+    """
+    Lints your Go files!
+    Concurrently runs a number of Go lint tools.
+    """
+
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
+        package='github.com/alecthomas/gometalinter', flag='-u')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    SEE_MORE = 'http://gopkg.in/alecthomas/gometalinter.v2'
+
+    def create_arguments(self, filename, file, config_file,
+                         enable: typed_list(str)=(),
+                         enable_all: bool=False,
+                         disable: typed_list(str)=(),
+                         gometalinter_config_file: str='',
+                         ignore_config: bool=False):
+        """
+        :param enable:
+            List of linters to enable.
+            Some linters are disabled by default. Refer goo.gl/gMirRv
+        :param enable_all:
+            Enable all supported linters.
+        :param disable:
+            List of linters to disable.
+        :param gometalinter_config_file:
+            A JSON configuration file for gometalinter.
+            It overrides ``gometalinter.json`` which is picked up by
+            default, if present.
+        :param ignore_config:
+            Ignores configuration files, if present.
+            Can't be used together with ``gometalinter_config_file``.
+        """
+        # Arguments are parsed in order.
+        args = (filename,)
+        if enable:
+            args += tuple('--enable={}'.format(e) for e in enable)
+        if enable_all:
```

Comment:
`enable` and `enable_all` should be mutually exclusive in my opinion. Same goes for `enable_all` and `disable`. If the user is adding any two of these settings we should throw an error.

Reply:
There are ~30 linters that are supported by gometalinter. ~18 of them are enabled by default.
- `enable_all` and `enable`:
There was a reason I left it like this. Can't remember it ATM. Will look into it surely. Plus, gometalinter supports this behavior.
- `enable_all` and `disable`:
For example:  In case you want to enable 25 of them, then you first need to enable them all and then disable the remaining 5. Same goes for `disable_all` and `enable`.

P.S. - Arguments are parsed in order.

Reply:
okay yes just confirm the reason for `enable_all` and `enable` 👍 
btw i don't see the `disable_all` setting

Reply:
I know; the PR is marked as WIP. :smile: